tests/test_config.py

```diff
@@ -187,3 +188,38 @@ def test_connections_toml_override_config_toml(test_snowcli_config, snowflake_ho
     assert CONFIG_MANAGER["connections"] == {
         "default": {"database": "overridden_database"}
     }
+
```

Comment:
Can we add one simple test case (probably in some other test suite) confirming that running an actual command will fail (not only config initialization)?